exe/genv-devices

```diff
@@ -240,8 +316,18 @@ def do_detach(devices: Devices, eid: str, index: Optional[int]) -> None:
 
 
 def do_ps(devices: Devices, format: str, header: bool, timestamp: bool) -> None:
+    """
+    Print genv devices data in a human-readable fashion to the stdio.
+    :param devices: dictionary describing all existing genv devices.
+    :param format: Which format should be used to print the devices data - csv or tui (default: tui).
+    :param header: a flag instructing if we should print a header as part of the output (default: print header).
+    :param timestamp: a flag instructing if we should print attachment time as is or relative duration (now - creation).
```

Comment:
also here

Reply:
Fixed, but adding a default value might be a good idea